packages/realm/scripts/generate-ios-libs.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+# Start in the root directory of the project.
+cd "$(dirname "$0")/.."
+
+# Take homebrew installs as well
+CMAKE_DIRECTORY=$(dirname "$CMAKE_PATH")
+export PATH="$CMAKE_DIRECTORY:$PATH"
```

Comment:
We should avoid referring to `CMAKE_PATH` until we need it (`BUILD_REALM_CORE` is set).